code/datums/weather/weather.dm

```diff
@@ -150,6 +152,7 @@
 /datum/weather/proc/wind_down()
 	if(stage >= WIND_DOWN_STAGE)
 		return
+	SEND_GLOBAL_SIGNAL(COMSIG_WEATHER_WINDDOWN(type))
 	stage = WIND_DOWN_STAGE
 	update_areas()
 	for(var/M in GLOB.player_list)
```

Comment:
there are tons of all players in game loops in weather code when we have clients_by_z_level already, theres several here and in SSweather

Reply:
Can't convert all of em, since the stuff in weather.dm relies on looking at all living mobs, which don't seem to have a similar list, but it's close enough